tests/CrudPanel/CrudPanelColumnsTest.php

```diff
@@ -0,0 +1,163 @@
+<?php
+
+namespace CrudPanel;
+
+class CrudPanelColumnsTest extends BaseCrudPanelTest
+{
+    /** @test */
+    public function it_can_add_column_by_name()
+    {
+        $expectedColumn = [
+            'name'  => 'column',
+            'label' => 'Column',
+        ];
+
+        $this->crudPanel->addColumn('column');
+
+        $this->assertEquals(1, count($this->crudPanel->columns));
+        $this->assertContains($expectedColumn, $this->crudPanel->columns);
+    }
+
+    /** @test */
+    public function it_can_add_column()
+    {
+        $expectedColumn = [
+            'name'  => 'column',
+            'label' => 'Column',
+        ];
+
+        $this->crudPanel->addColumn($expectedColumn);
+
+        $this->assertEquals(1, count($this->crudPanel->columns));
+        $this->assertContains($expectedColumn, $this->crudPanel->columns);
+    }
+
+    /** @test */
+    public function it_can_add_columns_by_name()
+    {
+        $expectedColumns = [
+            [
+                'name'  => 'column',
+                'label' => 'Column',
+            ],
+            [
+                'name' => 'column2',
+                'label' => 'Column2',
+            ],
+        ];
+
+        $this->crudPanel->addColumns(['column', 'column2']);
```

Comment:
This test makes an assumption about what the label should be (which I suppose is valid at the moment) but it seems somewhat brittle.